test/functional/wallet_transactiontime_rescan.py

```diff
@@ -158,5 +159,10 @@ def run_test(self):
                 assert_equal(tx['time'], cur_time + ten_days + ten_days + ten_days)
 
 
```

Comment:
Add some log message here? For example:
```suggestion
        self.log.info('Test handling of invalid parameters for rescanblockchain')
```

Reply:
Make sense, done!